From 9a4113fbbae65ee82d8cd6537963bc3d3b14bcca Mon Sep 17 00:00:00 2001
From: will-v-pi <108662275+will-v-pi@users.noreply.github.com>
Date: Mon, 7 Jul 2025 16:33:58 +0100
Subject: [PATCH] Lock picotool version on master to 2.1.1 (#2401)

The master SDK branch (and the SDK release tags eg 2.1.1) should point to specific picotool tags, to ensure you get a compatible picotool (note this is beyond the 2.1.1 tag in master, so will just fix this for users who check out the HEAD of master)
---
 tools/Findpicotool.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/Findpicotool.cmake b/tools/Findpicotool.cmake
index e5bfaeaac..4720a79f0 100644
--- a/tools/Findpicotool.cmake
+++ b/tools/Findpicotool.cmake
@@ -37,7 +37,7 @@ if (NOT TARGET picotool)
         message("Downloading Picotool")
         FetchContent_Populate(picotool QUIET
             GIT_REPOSITORY https://github.com/raspberrypi/picotool.git
-            GIT_TAG develop
+            GIT_TAG 2.1.1
 
             SOURCE_DIR ${picotool_INSTALL_DIR}/picotool-src
             BINARY_DIR ${picotool_INSTALL_DIR}/picotool-build
